hclgen/hcl_gen.go

```diff
@@ -0,0 +1,659 @@
+package hclgen
```

Comment:
For organization, this folder should live in the `internal/CLI` folder, rather than in its own top level folder. The file name should also be `hclgen.go`, no underscore. Like `internal/cli/datagen.go`